README.md

```diff
@@ -690,3 +690,146 @@ In general, `for of` is recommended over `for in` or `forEach`
 </details>
 
 ---
+
+###### 19. Array Manipulation: Understanding array.push() Method in JavaScript
+
+```javascript
+let arr = [1, 2, 3, 4];
+let result = arr.push(5);
+console.log(result);
+console.log(arr);
+```
+
+What does the above code snippet output?
+
+- A: 'result' will be 5 and 'arr' will be [1, 2, 3, 4, 5]
+- B: 'result' will be 5 and 'arr' will be [5, 1, 2, 3, 4]
+- C: 'result' will be 4 and 'arr' will be [2, 3, 4, 5]
+- D: 'result' will be 4 and 'arr' will be [5, 2, 3, 4]
+
+<details><summary><b>Answer</b></summary>
+<p>
+
+#### Answer: A
+
+The `array.push()` method in JavaScript adds one or more elements to the end of an array and returns the new length of the array. In the given code, arr is an array with the values `[1, 2, 3, 4]`. The `push()` method is called with the argument 5, which adds `5` to the end of the arr array.
+
+The `push()` method returns the new length of the array after the addition of the element(s). In this case, the new length of arr will be 5 because `5` is added to the end of the array.
+
+</p>
+</details>
+
+---
+
+##### 20. Array Manipulation: Understanding array.unshift() Method in JavaScript
+
+```javascript
+let arr = [3, 5, 7, 9];
+let result = arr.unshift(1, 2);
+console.log(result);
+console.log(arr);
+```
+
+What does the above code snippet output?
+
+- A: 'result' will be 6 and 'arr' will be [3, 5, 7, 9, 1, 2]
+- B: 'result' will be 6 and 'arr' will be [1, 2, 3, 5, 7, 9]
+- C: 'result' will be 4 and 'arr' will be [1, 2, 3, 5]
+- D: 'result' will be 4 and 'arr' will be [7, 9, 1, 2]
+
+<details><summary><b>Answer</b></summary>
+<p>
+
+#### Answer: B
+
+The `array.unshift()` method in JavaScript adds one or more elements to the beginning of an array and returns the new length of the array. In the given code, arr is an array with the values `[3, 5, 7, 9]`. The `unshift()` method is called with the arguments 1 and 2, which adds 1 and 2 to the beginning of the arr array.
+
+The `unshift()` method returns the new length of the array after the addition of the element(s). In this case, the new length of arr will be 6 because 1 and 2 are added to the beginning of the array, shifting the existing elements to the right.
+
+</p>
+</details>
+
+---
+
+##### 21. Array Manipulation: Understanding array.pop() Method in JavaScript
+
+```javascript
+const myArray = [1, 2, 3, 4, 5];
+const poppedValue = myArray.pop();
+console.log(poppedValue);
+```
+
+What does the above code snippet output?
+
+- A: 1
+- B: 5
+- C: 'undefined'
+- D: An error will occur
+
+<details><summary><b>Answer</b></summary>
+<p>
+
+#### Answer: B
+
+The `Array.pop()` method in JavaScript removes the last element from an array and returns that element. In this case, myArray is an array with elements `[1, 2, 3, 4, 5]`, and `myArray.pop()` is called, which removes the element 5 from the array and returns it.
```

Comment:
Please wrap variable name such as myArray in tick ex: `myArray` for better readability